po/mb_server.de.po

```diff
@@ -740,7 +740,7 @@ msgid "Add child"
 msgstr "Kind hinzufügen"
 
 #: ../root/cdstub/layout.tt:29
-msgid "Add disc ID to an existing release"
+msgid "Add medium ID to an existing release"
```

Comment:
The .po files are downloaded from Transifex, so any changes you make to them will be overwritten. :)

Let's revert all of these changes too, and not worry about changing anything under po/. We'll take care of extracting/updating the translations at release time.